app/components/chat/ModelSelector.tsx

```
@@ -165,32 +415,165 @@ export const ModelSelector = ({
+                          setModel(firstModel.name);
+                        }
+                      }
+
+                      setIsProviderDropdownOpen(false);
+                      setProviderSearchQuery('');
+                      setDebouncedProviderSearchQuery('');
+                    }}
+                    tabIndex={focusedProviderIndex === index ? 0 : -1}
+                  >
+                    <div
+                      dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)